packages/sitecore-jss-vue/src/components/RichText.test.ts

```diff
@@ -111,4 +112,85 @@ describe('<RichText />', () => {
     expect(rendered.attributes()).toMatchObject(attrs);
     expect(rendered.element.innerHTML).toBe(props.field.value);
   });
+
+  it('should initialize links', () => {
+    const props = {
+      field: {
+        value: '<div id="test"><h1>Hello!</h1><a href="/t10">1</a><a href="/t10">2</a></div>',
+      },
+    };
+    const rendered = mount(RichText, { props });
+    expect(rendered.element.innerHTML).toContain('<div id="test">');
+    expect(rendered.element.innerHTML).toContain('<h1>Hello!</h1>');
+    expect(rendered.element.innerHTML).toContain('<a href="/t10">1</a>');
+    expect(rendered.element.innerHTML).toContain('<a href="/t10">2</a>');
+  });
+
+  it('should not initialize links when does not have value', () => {
+    const props = {
+      field: {
+        value: '',
+      },
+    };
+    const rendered = mount(RichText, { props });
+    const hasChildNodes = rendered.element.hasChildNodes();
+    expect(hasChildNodes).toBe(false);
+  });
+  it('should not initialize links if no links in markup', () => {
```

Comment:
This test is not related to the links,  we have the same case covered in another test, can be removed
and if you check that you don't have `<a href='' /> ` -  it doesn't check that links are not initialized